src/lesson6_task1/LightTransport.java

```diff
@@ -0,0 +1,56 @@
+package lesson6_task1;
+
+
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+public class LightTransport extends LandTransport {
+    String bodyworkType;
+    int countPassengers;
+    float way;
+
+    //конструктор
+    public LightTransport(int powerHours, int maxSpeed, String brand, float weight, int countOfWheels, float fuelConsumption,
+                          String bodyworkType, int countPassengers) {
+        super(powerHours, maxSpeed, brand, weight, countOfWheels, fuelConsumption);
+        this.bodyworkType = bodyworkType;
+        this.countPassengers = countPassengers;
+        this.way = way;
+    }
+
+    //возвращает строку с описанием всех характеристик объекта
+    @Override
+    public String getDescription() {
+        this.technicalChar = "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
+                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
+                "-Количество колёс " + this.countOfWheels + ";\n-Расход топлива " + this.fuelConsumption + " л/100км;\n" +
+                "-Тип кузова - " + this.bodyworkType + ";\n-Кол-во пассажиров " + this.countPassengers + " человек.\n";
+        return this.technicalChar;
+    }
+
+    /* метод который будет принимать время и считать сколько километров проедет машина двигаясь с максимальной скоростью и
+     сколько топлива она израсходует за это время, результат вывести в консоль,
+     расчёт израсходуемого топлива вынести в отдельный метод private.
+ */
+    public void getWay() {
```

Comment:
@Nadya101  /* метод который будет  **### _ПРИНИМАТЬ_** время..
```suggestion
  
    public void getWay(float time) {
```

Reply:
Исправила тоже, но здесь сканер удалила. Заделала метод который принимает готовые значение _float time_